<launch>
    <!-- The ros nodes -->
    <node pkg="norfair_ros" name="converter"    type="converter.py"     output="screen"/>
    <node pkg="norfair_ros" name="tracker"      type="tracker.py"       output="screen"/>

    <!-- The parameter file -->
    <rosparam file="$(find norfair_ros)/config/norfair_darko.yaml"/>

</launch>